src/pyFAI/gui/widgets/FitParamView.py

```diff
@@ -411,3 +411,14 @@ def __constraintsClicked(self):
 
     def widgets(self):
         return [self.__labelWidget, self.__subLayout, self.__unit, self.__constraints]
+
+    def label(self):
+        """getter for the label"""
+        return self.__label
+
+    def setLabel(self, label):
+        """Change the label"""
+        if label != self.__label:
+            self.__label = label
+            self.__labelWidget.setText(f"{label}:")
+
```

Comment:
Would it make sense to modernise this to use @property / setter syntax instead of explicit setLabel()? Or is the current form required for Qt compatibility?